commercial/app/views/moneysupermarket/bestBuys.scala.html

```diff
@@ -6,7 +6,9 @@
             <div class="commercial__header">
                 <div class="commercial__header__inner">
                     <h3 class="commercial__title">
-                        <i class="i i-marque-54"></i><i class="i i-guardian-logo-commercial"></i><i class="i i-moneyshop-logo i-commercial-brand"></i>
+                        @fragments.inlineSvg("marque-54", "commercial")
+                        @fragments.inlineSvg("guardian-logo-commercial", "commercial")
```

Comment:
is this logo specific to commercial? if not, it could just go at top level (or maybe `logo`). 
if not, only needs to be called `guardian-logo` I should think?